backend/codesirius/internal_api/serializers/apikey.py

```
@@ -0,0 +1,22 @@
+from internal_api.models.apikey import APIKey
+
+logger = logging.getLogger(__name__)
+
+
+class APIKeySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = APIKey
+        fields = ["id", "name", "key", "is_active", "expires_at"]
+        read_only_fields = ["id", "key"]
+
+    def create(self, validated_data) -> APIKey:
```

Ruff (0.8.2)

Missing type annotation for function argument `validated_data`

(ANN001)